Dockerfile.dev

```diff
@@ -12,7 +12,9 @@ ENV \
 
 RUN apt-get update && apt-get install -y git && rm -rf /var/lib/apt/lists/*
 
-RUN pip install poetry==1.1.7
+RUN pip install --upgrade pip && \
+    pip install --upgrade poetry==1.1.13 && \
+    pip install --upgrade psycopg[binary]
```

Comment:
might be needed to add `pool` here too

```
psycopg[binary,pool]
```

Reply:
psycopg_pool should get pulled from the poetry config. It's just psycopg itself that has issues there.

Reply:
@bitner  Can `psycopg[binary]` be declared as a dependency in the published package? I was installing this pypgstac from source but running into:

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.8/dist-packages/psycopg/__init__.py", line 9, in <module>
    from . import pq  # noqa: F401 import early to stabilize side effects
  File "/usr/local/lib/python3.8/dist-packages/psycopg/pq/__init__.py", line 114, in <module>
    import_from_libpq()
  File "/usr/local/lib/python3.8/dist-packages/psycopg/pq/__init__.py", line 106, in import_from_libpq
    raise ImportError(
ImportError: no pq wrapper available.
Attempts made:
- couldn't import psycopg 'c' implementation: No module named 'psycopg_c'
- couldn't import psycopg 'binary' implementation: No module named 'psycopg_binary'
- couldn't import psycopg 'python' implementation: libpq library not found
```

I had to duplicate dependencies as I don't think the pip install from git likes the poetry.toml, but found I had to use `"psycopg[binary]==3.0.10",` to get past these errors. Does poetry support dependencies with extras?